companion/src/simulation/telemetrysimu.cpp

```diff
@@ -447,6 +447,27 @@ void TelemetrySimulator::generateTelemetryFrame()
   }
 }
 
+void TelemetrySimulator::startTelemetry()
+{
+  timer.start();
+}
+
+void TelemetrySimulator::stopTelemetry()
+{
+  timer.stop();
+
+  bool ok = (ui && ui->rssi_inst);
```

Comment:
Well there's not much point in setting a value here if you're not going to use it or actually care if the pointers are valid...  :)

Reply:
Sorry, just took your code without checking, stupid me!

Reply:
done

Reply:
why not write something like this here
```
if !ui || !ui->rssi_inst
    return;
```

it avoids the next 2 tests, or I forget something?